src/root_span_macro.rs

```diff
@@ -97,7 +97,12 @@ macro_rules! root_span {
             );
             std::mem::drop(connection_info);
 
-            $crate::root_span_macro::private::set_otel_parent(&$request, &span);
+            #[cfg(feature = "opentelemetry_0_13")]
+            $crate::otel_0_13::set_otel_parent(&$request, &span);
+
+            #[cfg(feature = "opentelemetry_0_14")]
+            $crate::otel_0_14::set_otel_parent(&$request, &span);
```

Comment:
This won't work in a third-party project using `tracing-actix-web` because those `set_otel_parent` functions are in a private module.
You should re-export them as public in the `private` mod below!

Reply:
Thanks. I changed it to re-export with versioned names since we can't have two definitions under the same name (the [features should be mutually exclusive](https://github.com/rust-lang/cargo/issues/2980), but there isn't a way to enforce that).